### PR TITLE
Removing Unused Variable Assignment

### PR DESCRIPTION
**Pull request by anjolaolubusi**

### CODE REVIEW

File: hexapod/ik_solver/recompute_hexapod.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def recompute_hexapod(dimensions, ik_parameters, poses):
     old_vector = vector_from_to(old_p1, old_p2)
     new_vector = vector_from_to(new_p1, new_p2)
 
-    translate_vector = vector_from_to(new_p1, old_p1)
+    #Removed unused assignment
```
Comment (mithi) [inline hexapod/ik_solver/recompute_hexapod.py:43]: Please remove this comment.

**Comment by mithi** [comment]
This looks good! Just remove the comment I mentioned.

Also, for this project, commit verbs should be in present case (and first letter capitalized). 

If your PR has multiple commit, be sure to squash it so that it's one commit.

There are some helpful links if you haven't done this before: 
- [squashing and rewording commits](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec)
- [Amend a commit on a Github pull request](https://www.burntfen.com/2015-10-30/how-to-amend-a-commit-on-a-github-pull-request)
- [To force or not to push](https://blog.sensible.io/2012/10/09/git-to-force-put-or-not-to-force-push.html)

If you have little experience with git, that's okay! you can create a fresh PR instead, it that's easier for you.

File: hexapod/ik_solver/recompute_hexapod.py (hunk 1 above)
Comment (anjolaolubusi) [reply hexapod/ik_solver/recompute_hexapod.py]: Done

**Comment by mithi** [comment]
Hi @anjolaolubusi, 

Thanks for trying to make a pull request, however, I think you need a little practice with using git 
(squashing and rewording).

And what I meant by present case is not "Removing" but "Remove", to be consistent with the other commit messages in this post.

I suggest you play around git first. And surely, you're next attempt to a PR would like be merged!

Other than the three links I have suggested above (which I highly recommend!), you might be interested in these as well:

- [Example Git Workflow](https://github.com/mithi/hexapod/wiki/Git-Workflow-guide)
- [Example Commit Style Guide](https://github.com/mithi/hexapod/wiki/Commit-Style-Guide)

**Pull request closed without merging**